arangod/RestHandler/RestVocbaseBaseHandler.cpp

```diff
@@ -553,18 +553,17 @@ void RestVocbaseBaseHandler::extractStringParameter(std::string const& name,
 
 std::unique_ptr<transaction::Methods> RestVocbaseBaseHandler::createTransaction(
     std::string const& collectionName, AccessMode::Type type,
-    OperationOptions const& opOptions) const {
+    OperationOptions const& opOptions, transaction::Options trxOpts) const {
```

Comment:
Should the options be passed by const reference? They mostly consist of booleans, but it seems they also carry the PeerState, which we probably don't want to copy unnecessarily.
```suggestion
    OperationOptions const& opOptions, transaction::Options const& trxOpts) const {
```

Reply:
The options are modified internally, but since they default to a default constructed temp value we cannot use a non-const ref. We could take them by rvalue ref though.